test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ClientValidatorCacheTest.cs

```diff
@@ -66,10 +62,7 @@ public void GetValidators_DoesNotCacheValidatorsWithIsReusableFalse()
             // Assert - 2
             Assert.NotSame(validators1, validators2);
 
-            Assert.Collection(
-                validators2,
-                v => Assert.Same(validators1[0], v), // Cached
-                v => Assert.NotSame(validators1[1], v)); // Not cached
+            Assert.Single(validators2.TakeWhile(v => validators1.Contains(v))); // only one validator is cached
```

Comment:
Might be easier to understand as
``` c#
    Assert.Contains(attribute1, validators2); // Cached
    Assert.NotContains(attribute2, validators2); // Not cached
```

Same for `GetValidators_DoesNotCacheValidatorsWithIsReusableFalse()` in the next file.